convert2rhel/main.py

```diff
@@ -209,7 +209,7 @@ def rollback_changes():
     utils.changed_pkgs_control.restore_pkgs()
     redhatrelease.system_release_file.restore()
     redhatrelease.yum_conf.restore()
-    subscription.rollback_renamed_repo_files()
+    subscription.rollback()
```

Comment:
The unregistering should happen before calling `utils.changed_pkgs_control.restore_pkgs()` three lines above, because that removes the installed subscription-manager.

Reply:
Fixed